runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/config/HttpBindingConfigTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.binding.http.internal.config;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+
+import org.junit.Test;
+
+import io.aklivity.zilla.runtime.engine.config.BindingConfig;
+import io.aklivity.zilla.runtime.engine.config.KindConfig;
+
+public class HttpBindingConfigTest
```

Comment:
Change this to `HttpQueryStringComparatorTest` to test `HttpQueryStringComparator` in isolation, and please name each method for its purpose, perhaps of the form `shouldCompare...EqualTo...()`, `shouldCompare...GreaterThan...()`, `shouldCompare...LessThan...()`.